packages/@okta/vuepress-site/guides/custom-url-domain/enable-the-custom-domain/index.md

```diff
@@ -39,19 +39,16 @@ Okta serves traffic over HTTPS (TLS) on your custom domain. Use this section to
 
 ### Create a CNAME Record for Your Subdomain
 Before Okta can serve traffic over your domain, you need to add an alias from your custom domain to the Okta subdomain of your Okta organization. You do this by creating or modifying a CNAME record for your custom domain name.
+    
+1. Return to your Domain Name registrar and locate the option to modify your DNS records.
+1. Create a new CNAME record and paste the value from the **Host** column into your registrar's **Name** or **Host** field.
```

Comment:
Will it always be a Record or Value field at their registrar's? Do we know for sure? If so, I'm okay leaving out "for example", otherwise, "for example" implies that it MAY be called Record or Value, but it MAY be called something else. Your thoughts?